build-system/tasks/performance/measure-documents.js

```diff
@@ -118,6 +151,23 @@ const readMetrics = page =>
     };
   });
 
+/**
+ * Adds analytics metrics
+ *
+ * @param {number} startTime
+ * @param {Array<number>} endTimes
+ * @param {object} metrics
+ * @return {object}
+ */
+function maybeAddAnalyticsMetric(startTime, endTimes, metrics) {
+  if (endTimes.length) {
+    const analyticsRequest =
+      endTimes.reduce((a, b) => a + b, 0) / endTimes.length - startTime;
```

Comment:
is `endTimes` a shared array? we only calculate the delay for the first request, when would it length be greater than 0.

Reply:
`endTimes` is an array that holds all the times that analytics requests gets fired. There are pages that use more than 1 analytics config, with each config having their own `visible` request, so we should account for these cases.

Reply:
In that case, it's possible that we are accounting for requests other than `visible` request. For example a timer request. 

IMO, first request would be sufficient to find regression, and less flaky.

Reply:
Ok, I am ok with that since we are averaging out 100 runs (default).